package.json

```diff
@@ -30,6 +30,7 @@
   "scripts": {
     "start": "node index.js",
     "dev": "nodemon index.js",
+    "build-js:analytics": "cp node_modules/@govuk-one-login/one-login-analytics/lib/analytics.js src/assets/javascript",
```

Comment:
If you want this to be included as part of the build, please don't copy this into the `src` folder, that will cause conflicts and/or can result in this file being committed into the repository accidentally.

Instead, the build process should be changed to produce a `build` folder, which is included as part of the packaging process.